italy.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"encoding/csv"
+	"io"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/gosimple/slug"
+)
+
+var italianRegions map[string]Record
+
+func getRegions() []Record {
+	f, err := os.Open("csv/italy/dpc-covid19-ita-regioni.csv")
+	if err != nil {
+		panic(err)
+	}
+	return parseItalianRegions(f)
+}
+
+func parseItalianRegions(reader io.Reader) []Record {
+	italianRegions = make(map[string]Record)
+	csvReader := csv.NewReader(reader)
+	records, _ := csvReader.ReadAll()
+
+	for i := 1; i < len(records); i++ {
+		provinceID := slug.Make(records[i][3]) // denominazione_regione
+
+		confirmed, _ := strconv.Atoi(records[i][10])                  // totale_attualmente_positivi
```

Comment:
Maybe the `confirmed` cases are the `totale_casi`?

Reply:
You're right, fixed. Midnight and csvs are not friends :)